bindings/java/iota-wallet-java/src/main/java/org/iota/types/account_methods/DecreaseNativeTokenSupply.java

```diff
@@ -0,0 +1,28 @@
+package org.iota.types.account_methods;
+
+import org.iota.types.TransactionOptions;
+import org.iota.types.ids.TokenId;
+
+/// Melt native tokens. This happens with the foundry output which minted them, by increasing it's
```

Comment:
```suggestion
/// Melt native tokens. This happens with the foundry output which minted them, by increasing its
```
That's also wrong in the other files outside of the java binding, can you update everywhere?

Reply:
I looked in the other bindings, I couldn't find it
Do you mean other Rust files?

Reply:
I get these two files ![image](https://user-images.githubusercontent.com/46689931/192237309-edcfa5f4-7b80-4094-81ca-75258c71e2c3.png)